src/js/components/Video/__tests__/Video-test.js

```diff
@@ -1,95 +1,172 @@
+    expect(tree).toMatchSnapshot();
+  });
 
-test('Video mute renders', () => {
-  const component = renderer.create(
-    <Grommet>
-      <Video mute>{CONTENTS}</Video>
-    </Grommet>,
-  );
-  const tree = component.toJSON();
-  expect(tree).toMatchSnapshot();
-});
+  test('mute renders', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Video mute>{CONTENTS}</Video>
+      </Grommet>,
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
 
-test('Video controls renders', () => {
-  const component = renderer.create(
-    <Grommet>
-      <Video controls="over">{CONTENTS}</Video>
-      <Video controls="below">{CONTENTS}</Video>
-    </Grommet>,
-  );
-  const tree = component.toJSON();
-  expect(tree).toMatchSnapshot();
-});
+  test('controls renders', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Video controls="over">{CONTENTS}</Video>
+        <Video controls="below">{CONTENTS}</Video>
+      </Grommet>,
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
 
-test('Video fit renders', () => {
-  const component = renderer.create(
-    <Grommet>
-      <Video fit="cover">{CONTENTS}</Video>
-      <Video fit="contain">{CONTENTS}</Video>
-    </Grommet>,
-  );
-  const tree = component.toJSON();
-  expect(tree).toMatchSnapshot();
-});
+  test('fit renders', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Video fit="cover">{CONTENTS}</Video>
+        <Video fit="contain">{CONTENTS}</Video>
+      </Grommet>,
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('Play and Pause event handlers', () => {
+    const { container } = render(
+      <Grommet>
+        <Video controls="below" playing={false}>
+          {CONTENTS}
+        </Video>
+      </Grommet>,
+    );
+    const videoContainer = document.querySelector('video');
+    fireEvent.play(videoContainer);
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.pause(videoContainer);
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('End event handler', () => {
+    const { container } = render(
+      <Grommet>
+        <Video controls="below">{CONTENTS}</Video>
+      </Grommet>,
+    );
+    // Need to fire play event to get video playing before we fire ended event.
+    const videoContainer = document.querySelector('video');
+    fireEvent.play(videoContainer);
+    fireEvent.ended(videoContainer);
+    expect(container.firstChild).toMatchSnapshot();
+  });
 
-// test('Video plays', () => {
-//   const component = mount(
-//     <Grommet>
-//       <Video controls='below'>{CONTENTS}</Video>
-//     </Grommet>
-//   );
-//   const button = component.find('button').at(0);
-//   console.log('!!!', button.debug());
-//   button.simulate('click');
-//   expect(component).toMatchSnapshot();
-// });
+  test('Menu Button', () => {
```

Comment:
Why is Menu Button related to Video components? It is not clear what this test is doing.

Reply:
Oh, I think I got it, a better description here will to use the same terminology of the documentations, so something like 
```suggestion
  test('Configure Menu Button', () => {
```
I noticed the documentation for it is missing from the website, so I filed an issue https://github.com/grommet/grommet/issues/4190